jupyterlab_caip_optimizer/src/components/suggest_trials/index.spec.tsx

```diff
@@ -1,3 +1,4 @@
+jest.setTimeout(100000);
```

Comment:
Looks kind of strange as the first line, would this still work in a beforeEach block in the top level of this test? (same for the other file)